go/couler/conversion/argo_workflow.go

```diff
@@ -0,0 +1,62 @@
+package conversion
+
+import (
+	wfv1 "github.com/argoproj/argo/pkg/apis/workflow/v1alpha1"
+	pb "github.com/couler-proj/couler/go/couler/proto/couler/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+const sequentialStepsTemplateSuffix = "main-template"
+
+func convertToArgoWorkflow(workflowPb *pb.Workflow, namePrefix string) (wfv1.Workflow, error) {
+	templates := []wfv1.Template{{Name: namePrefix + sequentialStepsTemplateSuffix}}
+	// TODO: Handle DAG tasks.
+	for _, step := range workflowPb.GetSteps() {
```

Comment:
here we just for-loop the steps, so we assume the steps are sorted well? how about dependencies?

Reply:
Yes, currently we are relying on the order. I left a TODO in the code. Steps with dependencies will be handled once we added the DAG conversion.